src/components/Admin/RobotTeamAssigner.tsx

```diff
@@ -27,7 +28,7 @@ export const RobotTeamAssigner = () => {
 						)
 						.map(([address, robot]) => (
 							<li key={address}>
-								{address}
+								{shortId(address) + ' '}
```

Comment:
```suggestion
								{shortId(address)}{' '}
```
This is the idiomatic react way of adding space.